src/LampHandler.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+
+function LampHandler() {
+  const [lamp, setLamp] = useState(false);
+
+  const handleClick = () => {
+    setLamp((current) => !current);
```

Comment:
Nice 🔥 Can't you just write?
```suggestion
    setLamp(!lamp);
```